e2e/README.md

```diff
@@ -0,0 +1,16 @@
+## Bootkube E2E Testing
+
+This is the beginnings of E2E testing for the bootkube repo using the standard go testing harness. To run the tests once you have a kubeconfig to a running cluster just execute:
+`go test -v --nodes=<number of nodes> --kubeconfig=<filepath> ./e2e/`
+
+The number of nodes is required so that the setup can block on all nodes being registered.
+
+## Roadmap
+
+Implement whatever is needed to finish porting all functionality from pluton tests
+
+## Requirements
+
+Tests can't rely on network access to the cluster except via the kubernetes api. So no hitting nodes directly just because you can. This will maximize future portability with other setup tools.
```

Comment:
Whatever is easiest I suppose. It will take a little work to setup the ability to SSH to each machine but it'd certainly make porting over the tests a bit easier.  

@aaronlevy What are your thoughts on the direction to take immediate destructive needs?